build_projects/dotnet-host-build/build.sh

```diff
@@ -107,6 +107,11 @@ then
     ulimit -n 1024
 fi
 
+# Clean old NuGet packages
+rm -rf $HOME/.local/share/NuGet/Cache
```

Comment:
these  paths should technically be quoted, but I'm fairly certain other things are going to break first if there's a space in the home directory.